README.md

```diff
@@ -31,19 +31,19 @@ Our [build server](http://teamcity.monogame.net/?guest=1) builds, tests, and pac
 
 We support a growing list of platforms across the desktop, mobile, and console space.  If there is a platform we don't support, please [make a request](https://github.com/MonoGame/MonoGame/issues) or [come help us](CONTRIBUTING.md) add it.
 
-- Desktop PCs
- * Windows Store Apps (8, 8.1 and 10)
- * Windows (OpenGL & DirectX)
- * Linux (OpenGL)
- * Mac OS X (OpenGL)
-- Mobile Devices
- * Android (OpenGL)
- * iOS (OpenGL)
- * Windows Phone (8, 8.1 and 10)
-- Consoles (for registered developers)
- * PlayStation 4
- * PlayStation Vita
- * Xbox One (both UWP and XDK)
+ * Desktop PCs
+   * Windows Store Apps (8, 8.1 and 10)
+   * Windows (OpenGL & DirectX)
```

Comment:
@cra0zy - I guess that is a point to make.

Do we list platforms (like iOS) or systems (like iPhone/iPad)?  In some cases it is both like PS4 is both a platform and a system.

Looking at Unity...

  https://unity3d.com/unity/multiplatform

They sort of mix and match as well...  Android and Android TV are two different platforms to them.  Same with iOS and tvOS.

What we can learn there is splitting them by category is best.

  - Desktop
  - Mobile
  - Consoles
  - Smart TVs
  - VR
 - Web

We're already starting to do that... so we're maybe on the right track here.